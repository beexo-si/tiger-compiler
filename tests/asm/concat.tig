let
    var s := concat(concat(concat(concat(concat(concat(concat(concat(concat(concat(concat((
		"Objetivo:\n"),
		"    Sokoban se origino en Japon. El objetivo del juego es usar la menor cantidad\n"),
		"    de pasos para llevar las cajas a las posiciones marcadas. Las cajas solo\n"),
		"    pueden ser empujadas hacia adelante. Solo se puede empujar una sola caja al\n"),
		"    mismo tiempo. Sokoban es un juego con reglas simples pero con muchos niveles\n"),
		"    de dificultad, algunos niveles puede tardarte varias horas o incluso dias.\n"),
		"Controles:\n"),
		"    Arriba, derecha, abajo, izquierda: movimiento\n"),
		"    D, Delete, Supr: deshacer ultimo movimiento\n"),
		"    R: reiniciar nivel\n"),
		"    Esc/Q: salir\n"),
		"")
in
    print(s);
    0
end
