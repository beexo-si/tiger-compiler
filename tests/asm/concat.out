bien!
yes!!
Objetivo:
    Sokoban se origino en Japon. El objetivo del juego es usar la menor cantidad
    de pasos para llevar las cajas a las posiciones marcadas. Las cajas solo
    pueden ser empujadas hacia adelante. Solo se puede empujar una sola caja al
    mismo tiempo. Sokoban es un juego con reglas simples pero con muchos niveles
    de dificultad, algunos niveles puede tardarte varias horas o incluso dias.
Controles:
    Arriba, derecha, abajo, izquierda: movimiento
    D, Delete, Supr: deshacer ultimo movimiento
    R: reiniciar nivel
    Esc/Q: salir
Return code: 0
